### PR TITLE
SBERDOMA-1005 Add feature flags support

### PR DESCRIPTION
**Pull request by toplenboren**
## Problem

We have partial features that are needed to be closed under feature flags

## Solution

Simple feature flags mechanism. 

## Usage

https://user-images.githubusercontent.com/33755274/129718209-084f5bca-370c-437f-936f-9d40c4bcd8c2.mov

### CODE REVIEW

File: apps/condo/pages/settings/index.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 import Head from 'next/head'
+import Error from 'next/error'
 import { Typography, Tabs, Tooltip, Col } from 'antd'
 import { TitleHeaderAction } from '@condo/domains/common/components/HeaderActions'
 import { PageContent, PageHeader, PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
 import { useIntl } from '@core/next/intl'
 import { OrganizationRequired } from '@condo/domains/organization/components/OrganizationRequired'
 import { BillingChooser } from '../../domains/billing/components/Settings/BillingChooser'
-
+import { FeatureFlagRequired } from '../../domains/common/components/containers/FeatureFlag'
```
Comment (dkoviazin) [inline apps/condo/pages/settings/index.tsx:10]: @condo/domains

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/common/components/containers/FeatureFlag.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
+// features should be added at local storage as 'featureName1,featureName2'
+
+import React, { ReactNode, useEffect, useState } from 'react'
+import { Modal, Switch } from 'antd'
+import useKeyboardShortcut from 'use-keyboard-shortcut'
+
+
+const _getEnabledFeatures = (): Array<string> => {
```
Comment (Dimitreee) [inline apps/condo/domains/common/components/containers/FeatureFlag.tsx:8]: Why underscore? New style detected

File: apps/condo/domains/common/components/containers/FeatureFlag.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            enabledFeatures.push(name)
+        }
+
+        localStorage.setItem('features', enabledFeatures.join(','))
+    }
+}
+
+/**
+ * Returns true if feature is found in localstorage, false otherwise
+ * @param name - name of the feature
```
Comment (Dimitreee) [inline apps/condo/domains/common/components/containers/FeatureFlag.tsx:37]: Redurant comments, u've got typescript for this:)

File: apps/condo/domains/common/components/containers/FeatureFlag.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+ */
+export const hasFeature = (name: string): boolean => {
```
Comment (Dimitreee) [inline apps/condo/domains/common/components/containers/FeatureFlag.tsx:39]: featureName: string, then comments not needed.)

File: apps/condo/domains/common/components/containers/FeatureFlag.tsx [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+/**
+ * Controller which allows to set feature flags in localstorage using non-trivial cheat-code interface
+ */
+export const FeatureFlagsController: React.FC = () => {
+
+    useKeyboardShortcut(['D', 'O', 'M', 'A'], () => showModal(), { overrideSystem: false })
+
+    const [isModalVisible, setIsModalVisible] = useState(false)
+
+    const featureFlagsConfig = ['billing', 'settings', 'se']
```
Comment (Dimitreee) [inline apps/condo/domains/common/components/containers/FeatureFlag.tsx:91]: Get from .env, then you don't need to change the code while new features added

**Review by Dimitreee — APPROVED** [review]

**Pull request merged**